lib/src/resources/movies_repository.dart

```diff
@@ -0,0 +1,7 @@
+import 'movies_api_provider.dart';
+import '../model/movies_model.dart';
+
+class MoviesRepository {
```

Comment:
This repository should also be implementing an interface

Reply:
done